src/writeData/components/telegrafInputPluginsConfigurationText/swap.conf

```diff
@@ -1,3 +1,3 @@
- Read metrics about swap memory usage
-[inputs.swap]]
- # no configuration
+# Read metrics about swap memory usage
+[[inputs.swap]]
```

Comment:
looks like the newline at the end of files misses files with no configuration?

Reply:
Fixed